src/main/java/de/zalando/aruha/nakadi/controller/SubscriptionController.java

```diff
@@ -55,52 +55,70 @@ public SubscriptionController(final SubscriptionDbRepository subscriptionReposit
 
     @RequestMapping(method = RequestMethod.POST)
     public ResponseEntity<?> createOrGetSubscription(@Valid @RequestBody final SubscriptionBase subscriptionBase,
-                                                     final Errors errors, final NativeWebRequest nativeWebRequest) {
-
-        if (!featureToggleService.isFeatureEnabled(HIGH_LEVEL_API)) {
+                                                     final Errors errors,
+                                                     final NativeWebRequest request) {
+        if (!featureToggleService.isFeatureEnabled(FeatureToggleService.Feature.HIGH_LEVEL_API)) {
```

Comment:
I also agree with Viktor regarding static imports, in most of cases they make code laconic. 
Also I think if every of us will just change what he doesn't like we will move into direction when we will just refactor each other's code without writing any new code.
For now I'm more than fine if you don't use static imports in the code you create (until we have a coding guideline rule for that), but I don't really see the sense to removing static imports everywhere in existing code.